ledger/sandbox/src/test/suite/scala/com/digitalasset/platform/sandbox/services/reset/ResetServiceIT.scala

```diff
@@ -174,6 +173,17 @@ final class ResetServiceIT
         }
       }
 
+      "clear out all garbage" in {
+        val state = new WeakReference(serverResource.value.sandboxState)
+        for {
+          lid <- fetchLedgerId()
+          _ <- reset(lid)
+        } yield {
+          System.gc()
```

Comment:
To the best of my understanding, this simply suggests the runtime to run garbage collection but it doesn't guarantee it will happen. Can this introduce flakiness in the test?

Reply:
Yes, but it seems our VM is happy with the "suggestion". If this fails when we upgrade Java, we can disable it and find another way to test it then.